### PR TITLE
Miscellaneous improvements

### PR DESCRIPTION
**Pull request by TacoTheDank**
### 1st commit
- Deleted obsolete files

### 2nd commit
- Update gradle wrapper to 6.8 using git bash

### 3rd commit
- Adjust codestyle to an .editorconfig

### 4th commit
- Update buildTools
- Clean up the build.gradle file a bit

### 5th commit
- Use getSystemService KTX extension

### 6th commit
- Use viewbinding

### 7th commit
- Enable minify to shrink app size

Built APK and tested. Everything worked fine as far as I could tell.

### CODE REVIEW

**Comment by mueller-ma** [comment]
I'll review this, but I haven't found time yet.

File: app/build.gradle [hunk 1]
```diff
@@ -3,47 +3,46 @@ apply plugin: 'kotlin-android'
...
     }
 
     lintOptions {
         abortOnError false
     }
+
     compileOptions {
-        targetCompatibility 1.8
-        sourceCompatibility 1.8
+        encoding "UTF-8"
```
Comment (mueller-ma) [inline app/build.gradle:28]: Why is this needed?

**Comment by mueller-ma** [comment]
I re-coded most of your commits, except 
- Adjust codestyle to an .editorconfig: I'd like to keep the code style in the .idea files and therefore do not exclude them from the repo.
- Enable minify: I heard of crashes appearing when enabling minify, so I'd like make the next release without this one.

File: app/build.gradle (hunk 1 above)
Comment (TacoTheDank) [reply app/build.gradle:28]: It helps keep files from possibly being loaded in the wrong encoding, particularly for Java 8.

**Comment by TacoTheDank** [comment]
> I re-coded most of your commits, except
> 
> * Adjust codestyle to an .editorconfig: I'd like to keep the code style in the .idea files and therefore do not exclude them from the repo.

Fair enough 👍 

> * Enable minify: I heard of crashes appearing when enabling minify, so I'd like make the next release without this one.

Ok with me. I added the -dontobfuscate to the proguard file so that debugging is easy for this purpose. It vastly shrinks the app's installation size on the device but still allows for easy debugging.

**Comment by mueller-ma** [comment]
With [minify enabled ](https://github.com/mueller-ma/Coffee/commit/89de65f9e26509be9c277c7ef03e9d9cf7f9bb65)I should have picked all commit from this PR. Thanks for your input.

**Comment by TacoTheDank** [comment]
Alright, thanks :)
